pkg/app/server.go

```diff
@@ -86,6 +90,16 @@ func RunKubeStateMetricsWrapper(ctx context.Context, opts *options.Options) erro
 func RunKubeStateMetrics(ctx context.Context, opts *options.Options) error {
 	ksmMetricsRegistry := prometheus.NewRegistry()
 	ksmMetricsRegistry.MustRegister(versionCollector.NewCollector("kube_state_metrics"))
+
+	clientGoVersion := promauto.With(ksmMetricsRegistry).NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "kube_state_metrics_client_go_info",
+			Help: "An info metric for the client-go version used by kube-state-metrics",
```

Comment:
Can we mark it as experimental?

Reply:
I assume this suggestion is now outdated. Do I need to update information on the `kube_state_metrics_build_info`  metric?